Cargo.toml

```diff
@@ -16,3 +16,6 @@ path = "src/lib.rs"
 [[bin]]
 name = "xml-analyze"
 path = "src/analyze.rs"
+
+[dependencies]
```

Comment:
This brings `lazy_static` dependency to the library directly. I'd like to avoid that. If you find a way to make it only usable for tests (I think it's dev-dependencies but I might be wrong), then I'm okay with merging.